src/SummaryRow.tsx

```diff
@@ -29,6 +32,7 @@ function SummaryRow<R, SR>({
       aria-rowindex={ariaRowIndex}
       className={`rdg-row rdg-row-${rowIdx % 2 === 0 ? 'even' : 'odd'} rdg-summary-row`}
       style={{ bottom }}
+      onClick={onClick}
```

Comment:
I accidentally deleted your comment

> Why this change? I don't think we expose SummaryRow?
So we can commit the value when summary row is clicked. We need to decide on when to commit the values